chocolatey-community/chocolatey-coreteampackages
# Add ffmpeg-full and update ffmpeg

## Description
- Adds package `ffmpeg-full` to install the `full build` of ffmpeg, whereas the `ffmpeg` package just installs the `essentials build`
- update descriptions of `ffmpeg` package to note that this is just installing the `essentials build`
- update the `update.ps1` script to use [this version info](https://www.gyan.dev/ffmpeg/builds/release-version) to check for the latest version (bugfix)

## Motivation and Context
For video heavy work environments it is helpful to have the full ffmpeg build with support for all possible codecs and features.
The maintainer of the remote decided to remove the build date from the version, so installing older versions with the same script is expected to fail.

## How Has this Been Tested?



- Set up this [chocolate-test-environment](https://github.com/majkinetor/chocolatey-test-environment)
- did test for install, uninstall and downgrade two versions back (that as much is provided by remote)
- Install/uninstall succeded
- `--version` fails, as expected


## Types of changes


## Checklist:


- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The changes only affect ~~a single package~~ two single packages (not including meta package).
